spec/models/manageiq/providers/kubernetes/container_manager/refresher_spec.rb

```diff
@@ -69,6 +69,7 @@ def assert_table_counts
     expect(ContainerImage.count).to eq(3)
     expect(ContainerImageRegistry.count).to eq(1)
     expect(ContainerComponentStatus.count).to eq(3)
+    expect(PersistentVolume.count).to eq(1)
```

Comment:
@cben I added the `assert_specific_persistent_volume` function including a check for `@ems.persistent_volumes.count` inside it. Although it's a heavy test here, I'm in favor of keeping it, as this case shows that second and third runs are not necessarily identical.